src/gridList.test.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import GridList from './GridList';
+
+const JSX_GridListStatic = (
+	<GridList
+		columns={{
+			default: 3
+		}}
+		items={[
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>
+		]}
+	/>
+);
+
+const responsiveColsDefault = 2,
+	responsiveColsMedium = 4,
+	responsiveColsLarge = 6;
+
+const JSX_GridListResponsive = (
+	<GridList
+		columns={{
+			default: responsiveColsDefault,
+			medium: responsiveColsMedium,
+			large: responsiveColsLarge
+		}}
+		items={[
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>,
+			<div>GridItem</div>
+		]}
+	/>
+);
+
+let gridList,
+	gridListNode;
+
+describe('Static GridList', function() {
+	beforeEach(() => {
+		gridList = TestUtils.renderIntoDocument(JSX_GridListStatic);
+		gridListNode = ReactDOM.findDOMNode(gridList);
+	});
+	afterEach(() => {
+		gridList = null;
+		gridListNode = null;
+	});
+
+	it('exists', function() {
+		expect(gridListNode).not.toBeNull();
+	});
+
+	it('wraps gridList items with element containing className "gridList-item"', function() {
+		const glItems = TestUtils.scryRenderedDOMComponentsWithClass(gridList, 'gridList-item');
+		expect(glItems.length).not.toBe(0);
+	});
+});
+
+describe('Responsive GridList', function() {
+	let glClassList;
+
+	beforeEach(() => {
+		gridList = TestUtils.renderIntoDocument(JSX_GridListResponsive);
+		gridListNode = ReactDOM.findDOMNode(gridList);
```

Comment:
Instead of `findDOMNode` we've been trying to use these the TestUtils functions like https://facebook.github.io/react/docs/test-utils.html#findRenderedDOMComponentWithClass for these kinds of tests, where we can. There are definitely still instances where you can't get away from avoiding findDOMNode, but since we are testing the classes components directly applied to the component, this should work. You can do something like

```
const gridList = TestUtils.findRenderedDOMComponentWithClass(gridList, GridList);
const gridListClassList = gridList.props.className;
```